resources/roca_lib.brs

```diff
@@ -123,16 +125,22 @@ end function
 ' Creates a test case with a given description.
 ' @param description a string describing this test case
 ' @param func the function to execute as part of this test case
-sub __it(description as string, func as object)
-    m.__suite.__registerCase("default", description, m.__suite, func)
+sub __it(description as string, func as object, args = invalid as dynamic)
+    m.__suite.__registerCase("default", description, m.__suite, func, args)
 end sub
 
-sub __fit(description as string, func as object)
-    m.__suite.__registerCase("focus", description, m.__suite, func)
+sub __fit(description as string, func as object, args = invalid as dynamic)
+    m.__suite.__registerCase("focus", description, m.__suite, func, args)
 end sub
 
-sub __xit(description as string, func as object)
-    m.__suite.__registerCase("skip", description, m.__suite, func)
+sub __xit(description as string, func as object, args = invalid as dynamic)
+    m.__suite.__registerCase("skip", description, m.__suite, func, args)
+end sub
+
+sub __it_each(argsArray as object, descriptionGenerator as object, func as object)
```

Comment:
Would you mind adding `__fit_each` and `__xit_each` as parallels to `__fit` and `__xit`?  It'd allow all the tests from `m.it_each` to be focused or skipped without having to wrap them in `fdescribe()` or `xdescribe()`!